mavros_msgs/msg/HilGPS.msg

```diff
@@ -0,0 +1,17 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_GPS
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_GPS
+
+std_msgs/Header header
+uint8 fix_type
+geographic_msgs/GeoPoint geo
```

Comment:
@vooon you were the first to suggest this. Then you saw the AMSL thing. But ok.
I will chang this later have stuff to do now